### DISCUSSION TITLE
Finding out which .env file was loaded?

### DISCUSSION THREAD

**Discussion by pke** [motdotla/dotenv#731] [Q&A]
I know its not recommended to have multiple .env files but I'd still like to know if its possible to get the path to the loaded .env file?

**Comment by motdotla** [comment]
You can set the path. Otherwise, the path is always the current working directory of the running process.

**Reply by motdotla** [reply]
https://github.com/motdotla/dotenv#path

**Reply by pke** [reply]
Yes I saw I can set it, but if I don't then dotenv uses always the .env file in the current directory. But what if the process used `ENVFILE` to designate the file to load? I guess then I could just query `env.ENVFILE` to find out which file was used?

**Reply by motdotla** [reply]
It's an interesting thought. We'll keep it in mind, but right now I think it adds too much to the library. Instead, for now you can construct the path using your own ENVFILE argument.
